getting_started/build-guide/build-from-scratch.rst

```diff
@@ -221,13 +225,26 @@ default values missing from it:
    ./ct-ng oldconfig V=1
    diff -u config-apl-gcc10.2-gdb9 .config
 
-"Install" toolchains by copying them to ``$SOF_WORKSPACE``.
+While other steps take minutes at most, building all toolchains may last
+about an hour depending on the performance of your system. Run this loop
+to build all toolchains without interruption:
+
+.. code-block:: bash
+
+   time for i in config*gcc10*; do
```

Comment:
we have two gcc10 config for apl:
```
config-apl-gcc10.2-gdb8.3
config-apl-gcc10.2-gdb9
```
Will this going to cause issue?

Reply:
lets use gdb9

Reply:
> we have two gcc10 config for apl:

Great catch, thanks.

> Will this going to cause issue?

I think I got away with it because globbing is "sorted according to the collating sequence in effect in the current locale."  https://pubs.opengroup.org/onlinepubs/009604499/utilities/xcu_chap02.html#tag_02_13_03

> let's use gdb9

Done, changed to `config*gcc10.2-gdb9`